src/plots/gl2d/scene2d.js

```diff
@@ -332,6 +337,8 @@ proto.destroy = function() {
     this.container.removeChild(this.svgContainer);
     this.container.removeChild(this.mouseContainer);
 
+    this.fullData = null;
+    this._inputs = null;
```

Comment:
@etpinard I added `this.fullData` and `this._inputs` to retain info prerequisite for the event data contents you suggested (below), please let me know if there's a better way.
```
this.graphDiv.emit('plotly_click', {
                points: [{
                    x: ,
                    y: ,
                    curveNumber: ,
                     pointNumber: ,
                    data: nextSelection.trace._input,
                    fullData: nextSelection.trace,
                    xaxis: this.axis,
                   yaxis: this.yaxis
                }]
```